tools/ARIAtools/extractProduct.py

```diff
@@ -753,6 +751,10 @@ def main(inps=None):
     else:
         demfile=None ; Latitude=None ; Longitude=None
 
+    # Load or download mask (if specified).
```

Comment:
Any specific reason why order is swapped with dem check?

Reply:
this is important because my code uses the DEM to resample the NLCD product.

Reply:
This might be problematic for ariaExtract.py as dem is not a required input. What do you need? Proj and transformation?

Reply:
If you need just the projection, why not pass it from the "merged_productbbox" function (see line 343 of extractProduct.py)? It is one of those functions outputs

Reply:
Plus to warp it to have the same expected dimensions/resolution as the dem, you could pass a lot of the same variables as we do from "merged_productbbox" to "prep_dem" (e.g. see the way "arrshape" from "merged_prodduct" is used in the gdal.Warp commands in "prep_dem")